recipes/python-toolchain/activate.bat

```diff
@@ -0,0 +1 @@
+set PIP_CONFIG_FILE="%LIBRARY_PREFIX%\etc\pip.conf"
```

Comment:
Usually it's `set "NAME=VALUE"`. No idea why, but using it that way prevented bugs in cmder